br/pkg/utils/key.go

```diff
@@ -89,3 +95,29 @@ func CompareEndKey(a, b []byte) int {
 
 	return bytes.Compare(a, b)
 }
+
+// encode
+func FormatAPIV2Key(key []byte, isEnd bool) []byte {
+	if isEnd && len(key) == 0 {
+		return []byte{APIV2KeyPrefixEnd}
+	}
+	apiv2Key := []byte{APIV2KeyPrefix}
+	return append(apiv2Key, key...)
+}
+
+// conversion between formated APIVersion key and backupmeta key
```

Comment:
```suggestion
// ConvertBackupConfigKey converts between formated APIVersion key and backupmeta key
```
It is recommended to comment every function in the style as "`Function-Name` does ...". Please refer to [here](https://go.dev/doc/effective_go#commentary).

Moveover, please add unit tests for new functions.

Reply:
done